include/uv-unix.h

```diff
@@ -267,6 +267,7 @@ typedef struct {
   int delayed_error;                                                          \
   int accepted_fd;                                                            \
   void* queued_fds;                                                           \
+  int chunk_read_size;                                                        \
```

Comment:
the interface change is not intentional - the requirement is to have a mechanism to associate the modified chunk size with a specific socket / stream upon API invocation, and use that value while reading in uv__write (write is inherently capable of writing as much as it can without blocking, so no issues). Do you have any thoughts on achieving this without introducing an interface change - either to stream / tcp structures?